resources/assets/js/components/UptimeMonitor.vue

```diff
@@ -0,0 +1,72 @@
+<template>
+    <grid :position="grid">
+        <section v-if="offlineUrls.length === 0">
+            <div class="internet-connection__icon">
+            </div>
+        </section>
+
+        <section v-if="offlineUrls.length">
+            <ul>
+                <li v-for="offline in offlineUrls">
+                    <h2>{{ offline.url }}</h2>
+                    <div>{{ offline.startedFailingAt | formatDuration }}</div>
+                </li>
+            </ul>
+        </section>
+    </grid>
+</template>
+
+<script>
+    import echo from '../mixins/echo';
+    import Grid from './Grid';
+    import moment from 'moment';
+    import duration from 'moment-duration-format';
+
+    export default {
+
+        components: {
+            Grid,
+        },
+
+        filters: {
+            formatDuration(start) {
+                return moment.duration(moment().diff(start), 'milliseconds').format("d[d] h[h] m[m]");
```

Comment:
Add this functionality to a new function in `helpers.js` and add some tests to make sure it works properly.